modules/update-common-terms.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+
+= Common Terms
+
+Control Plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines. 
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service to view valid updates and update paths based on your current component versions and information in the graph.
```

Comment:
Because it is first instance here, you might consider adding the acronym (even though you do the same in the OSUS entry later, which makes sense given that it's a definition list):

```suggestion
Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service (OSUS) to view valid updates and update paths based on your current component versions and information in the graph.
```